Localization/de-DE/home.yaml

```
@@ -41,6 +53,49 @@ Für technische Hilfe: ",
 #URLs - Facebook and Twitter have localised name in certain languages:
 "foldingforum.org",
 "Facebook",
-"Twitter"
+"Twitter",
+
+#LATEST POSTS bold text
+#"LATEST POSTS",
+"NEUESTE ARTIKEL",
+
+# WHY WE NEED YOU bold text
+#"WHY WE NEED YOU",
+"WARUM WIR SIE BRAUCHEN",
+
+# Associated text
+#"Folding@home is a project focused on disease research. The problems we're solving require so many computer calculations and we need your help to find the cures!",
+"Folding@home ist ein Projekt, das sich auf die Erforschung von Krankheiten konzentriert. Die Problemlösungen erfordern eine so große Menge an Berechnungen, dass wir Ihre Hilfe brauchen, um Heilmittel zu finden!",
+
+#FIGHT DESIEASES TOGETHER WITH US bold text
+#"FIGHT DISEASES TOGETHER WITH US.",
+"BEKÄMPFT KRANKHEITEN GEMEINSAM MIT UNS",
+
+#Associated text
+#"The Folding@home software allows you to share your unused computer power with us so that we can research even more potential cures.",
+"Die Folding@home Software erlaubt es Ihnen, uns Ihre ungenutzte Rechenleistung zur Verfügung zu stellen, so dass wir noch mehr mögliche Heilmittel untersuchen können.",
```

```suggestion
"Die Folding@home-Software erlaubt es Ihnen, uns Ihre ungenutzte Rechenleistung zur Verfügung zu stellen, so dass wir noch mehr mögliche Heilmittel untersuchen können.",
```

---

```suggestion
"Die Folding@home Software erlaubt es Ihnen, uns Ihre ungenutzte Rechenleistung zur Verfügung zu stellen, so dass wir noch mehr mögliche Therapien untersuchen können.",
```
Imho 'potential cures' is more _Therapien_ than _Heilmittel_ in this context